### ISSUE TITLE
Using the optimizer, I get the error "must be str, not Prefetch"

### ISSUE THREAD

**Issue by elmehalawi** [tfoxy/graphene-django-optimizer#9]
I have some django models:

```python
class UserWorkspace(SafeDeleteModel):
    name = models.CharField(max_length=50)
    user = models.ForeignKey(User, on_delete=models.CASCADE)

    instance_groups = models.ManyToManyField(InstanceGroup)


class InstanceGroup(SafeDeleteModel):
    instances = models.ManyToManyField(project)


class project(SafeDeleteModel):
    """
    Projects
    """
    uploaded_file = models.ForeignKey('primavera.UploadedFile', on_delete=models.CASCADE)
    proj_id = models.IntegerField(null=True, help_text="FK to PROJECT table - identifies the project")
    # ...........................


class task(SafeDeleteModel):
    """
    Activities
    """
    uploaded_file = models.ForeignKey('primavera.UploadedFile', on_delete=models.CASCADE)
    task_id = models.IntegerField(null=True, help_text="FK to TASK table - identifies the task")
    proj_id = models.ForeignKey(null=True, help_text="FK to PROJECT table - identifies the project", to="project", on_delete=models.CASCADE, related_name="tasks")
    # ............................
```

The query looks like this:

```python
def resolver_for_model(model):
    return lambda root, info, **kwargs: gql_optimizer.query(model.objects.all(), info)

class BaseQuery():
    all_user_workspace = graphene.List(type_for_model(UserWorkspace), resolver=resolver_for_model(UserWorkspace))
```

When I run this query:

```gql
{
  allUserWorkspace {
    id
    name
    instanceGroups {
      instances {
        lastRecalcDate
        id
        tasks {
          id
        }
      }
    }
  }
}
```

I get this error:

> graphql.error.located_error.GraphQLLocatedError: must be str, not Prefetch

I looked at that code and tried to replicate the problem by running something like this:

```python
Instance.objects.all().prefetch_related(Prefetch('tasks'))
```

This works just fine though. I'm not sure what could be causing this error.

**Comment by elmehalawi** [comment]
Is there a way to somehow introspect the query that the library is generating?

**Comment by tfoxy** [comment]
Hi! Sorry for the long delay. I was on vacation.

I was thinking of a way to introspect the query. In the following days I will release a new version with that feature. After doing that, I will reproduce the bug and fix it.

**Comment by tfoxy** [comment]
Hi! Sorry for the long time. Its solved in v0.3.6!

**Comment by elmehalawi** [comment]
Thank you very much!
